integration-tests/src/test/kotlin/org/stellar/anchor/platform/KotlinReferenceServerIntegrationTest.kt

```diff
@@ -0,0 +1,93 @@
+package org.stellar.anchor.platform
+
+import io.ktor.http.*
+import kotlinx.coroutines.runBlocking
+import org.junit.jupiter.api.*
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.skyscreamer.jsonassert.JSONAssert
+import org.stellar.anchor.api.callback.SendEventRequest
+import org.stellar.anchor.util.StringHelper.json
+import org.stellar.reference.client.AnchorReferenceServerClient
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+internal class KotlinReferenceServerIntegrationTest {
+  private val testProfileRunner =
+    TestProfileExecutor(
+      TestConfig("sep24").also {
+        it.env["run_all_servers"] = "false"
+        it.env["run_kotlin_reference_server"] = "true"
+        it.env["run_docker"] = "false"
+      }
+    )
+
+  @BeforeAll
+  fun setup() {
+    testProfileRunner.start()
+  }
+
+  @AfterAll
+  fun destroy() {
+    testProfileRunner.shutdown()
+  }
+
+  @Test
+  fun `test if the reference server records the events sent by sendEvent() method`() {
```

Comment:
Are we trying to verify that the contents of the callback is correct? Shouldn't this be captured by a test in the `platform` project instead?

Reply:
This is part of the integration test and it is testing the reference server's correctness of recording the events from the `/event` endpoint. The `platform` project currently does not run integration tests.